### ISSUE TITLE
After 7.1.2 upgrade, Raw variable format is returning undefined, not working as expected. 

### ISSUE THREAD

**Issue by mavasaf** [grafana/grafana#26819]
**What happened**:
Template Variables Format : template variables used in the query editor with raw format returning undefined. 

**What you expected to happen**:
Expecting the template variables to give the raw text in the underlying selection. However, it returns undefined. 

**How to reproduce it (as minimally and precisely as possible)**:
1. Have a template variable created in the dashboard. 
*variable name* $Operator
*variable Custom value* "AND" , "AND NOT" , "OR"
2. reference the template variable with the variable format as mentioned [here](https://grafana.com/docs/grafana/latest/variables/advanced-variable-format-options/#raw)
Eg: ${Operator:raw}

*Expected result :* 
AND ( when selected AND)
AND NOT ( when selected AND NOT)
OR ( when selected OR) 

Current Result : **undefined**

**Anything else we need to know?**:

**Environment**:
- Grafana version: 7.1.2
- Data source type & version: Elasticsearch 
- OS Grafana is installed on: Windows
- User OS & Browser: Windows , Google Chrome
- Grafana plugins:
- [Others:]

**Comment by mavasaf** [comment]
![image](https://user-images.githubusercontent.com/68710456/89454994-70cda880-d716-11ea-8e4e-99c644f6348c.png)
![image](https://user-images.githubusercontent.com/68710456/89455001-73c89900-d716-11ea-940f-b6c34326ff93.png)

**Comment by mavasaf** [comment]
Please note this was working well until yesterday in 7.1.1.
Issue started to occur only after 7.1.2 upgrade.

**Comment by zoltanbedi** [comment]
Thanks for reporting this @mavasaf. I'm going to close this issue as this is a duplicate of #26816
